File: benchmark/comparison/results/blind_ts/06.ts
function repeat(s: string, n: number, sep: string): string {
  if (n === 0) return "";
  const parts: string[] = [];
  for (let i = 0; i < n; i++) parts.push(s);
  return parts.join(sep);
}

console.log(repeat("ha", 3, " "));
console.log(repeat("ab", 1, ","));
console.log(repeat("x", 0, ","));
console.log(repeat("hi", 2, "-"));

File: benchmark/comparison/results/blind_py/06.py
def repeat(s, n, sep):
    if n == 0:
        return ""
    return sep.join([s] * n)

print(repeat("ha", 3, " "))
print(repeat("ab", 1, ","))
print(repeat("x", 0, ","))
print(repeat("hi", 2, "-"))
